app/assets/stylesheets/components/insights/c-article.scss

```diff
@@ -108,6 +108,8 @@
         display: block;
         margin: 1em 0;
         padding-left: 20px;
+        line-height: 1.38;
```

Comment:
Usually, we use a global line height set in `settings.scss`. Doesn`t it work for you?

Reply:
@davidsingal it looks like globals are defined in `_typography.scss` but this does not include for list items. I added at the component level because I wasn't sure if this was a global pattern. Happy to amend to what you think is best?

Reply:
So feel free to add it in global styles.